pom.xml

```diff
@@ -45,7 +45,7 @@
         <formatterConfigPath>formatter.xml</formatterConfigPath>
         <settingsFile>${java.io.tmpdir}/effective-settings.xml</settingsFile>
         <mtf.tools.version>1.2.0</mtf.tools.version>
-        <mtf.javaopts></mtf.javaopts>
+<!--        <mtf.javaopts></mtf.javaopts>-->
         <jacoco.version>0.8.10</jacoco.version>
```

Comment:
You can use the newer one [0.8.12](https://mvnrepository.com/artifact/org.jacoco/jacoco-maven-plugin/0.8.12)

Reply:
done